### DISCUSSION TITLE
image won't show on website

### DISCUSSION THREAD

**Discussion by strawbs1989** [community/community#123516] [Programming Help] [answered]
### Select Topic Area

Question

### Body

hi team basically i'm uploading images to my site on github 
and it's not showing on my github server the image is showing as blank
yes its a png file see here [https://github.com/strawbs1989/CoolvibesR/blob/main/socialicons/fb.png](url)
but DO NOT know why it keeps going blank. could someone possible help

**Comment by davevad93** [comment]
Hi @strawbs1989 ,

Try downloading that same image from your repository, if you can't open it either there must be a problem with the file itself being corrupt.  Make sure the file is ok before uploading it, try to change the filename extension from `png` to `jpeg` for example and check if that fixes it.

**Reply by strawbs1989** [reply]
that worked ta

**Reply by davevad93** [reply, marked as answer]
You're welcome @strawbs1989 , if you found my answer useful please, mark it as `answered`. Have a nice day!

**Comment by strawbs1989** [comment]
OK thanks will do
